public/Aframe/samples/floor.js

```diff
@@ -0,0 +1,20 @@
+AFRAME.registerComponent("floor", {
```

Comment:
Can you remove the components that are unrelated to the numpad from this PR @amoliksingh ?

Reply:
Removed